lib/CryptoNoteCore/Currency.cpp

```diff
@@ -141,7 +141,7 @@ bool Currency::getBlockReward(
     uint32_t height,
     uint64_t blockTarget) const
 {
-    assert(alreadyGeneratedCoins <= m_moneySupply);
+    //assert(alreadyGeneratedCoins <= m_moneySupply);
```

Comment:
is there a reason for this? unsure of how his relates to Breakpad

Reply:
True, this one got in this PR by an accident after rebasing on top of some older changes.

Reply:
Reverted this unrelated changes in a new commit.